junction/templates/profiles/userprofile.html

```diff
@@ -17,7 +17,7 @@
 
 {% block breadcrumbs %}
     {{ block.super }}
-    {% breadcrumb "Dashboard" "user-proposals-list" %}
+    {% breadcrumb "Dashboard" "/profiles" %}
     {% breadcrumb "Profile" "user-profile"%}
```

Comment:
Given the second param is dummy, and of no use can you add a comment regarding that?

Reply:
sure, will work on that.

Reply:
Done.